ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/numberinput/v1/numberinput/numberinput.html

```diff
@@ -49,6 +49,7 @@
             disabled="${!numberinput.enabled}"
             readonly="${numberinput.readOnly}"
```

Comment:
Do we required this readOnly attribute in all components, as you've removed it in radiobutton.

Reply:
readonly attribute is not supported on radio buttons. Radio buttons allow users to select one option from a group of options. Making individual radio buttons readonly doesn't make sense because the purpose of radio buttons is to allow selection. Making them readonly will not serve the purpose.